cluster-state-service/handler/store/datastore.go

```diff
@@ -158,7 +181,17 @@ func (datastore etcdDataStore) stream(ctx context.Context, keyPrefix string, kvC
 			}
 			resetStreamIdleTimer(streamIdleTimer)
 			for _, ev := range event.Events {
-				kv := map[string]string{string(ev.Kv.Key): string(ev.Kv.Value)}
+				// Skip empty events, such as Etcd deletes.
```

Comment:
Shouldn't a stream technically notify user of deletes? I'm not sure how yet, but it may be useful info to emit to users. What do you think? Maybe add a TODO to deal with it?